<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ToolWindow</class>
 <widget class="QWidget" name="ToolWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>158</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RWR Window Tool</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0,0,0,0">
     <item row="4" column="1" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0">
       <item>
        <widget class="QLabel" name="fullscreenLabel">
         <property name="text">
          <string>Fullscreen: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="resolutionLabel">
         <property name="text">
          <string>Resolution: </string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="5" column="1" colspan="2">
      <widget class="QTextBrowser" name="textBrowser">
       <property name="focusPolicy">
        <enum>Qt::FocusPolicy::NoFocus</enum>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Microsoft YaHei UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Reading game graphics settings...&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QRadioButton" name="borderlessRadioButton">
       <property name="text">
        <string>Borderless fullscreen</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QRadioButton" name="windowRadioButton">
       <property name="text">
        <string>Better window</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="rwrConfigureButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>RWR Configure</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="rwrStartButton">
         <property name="text">
          <string>Start RWR</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
